pkg/ansible/ansible.go

```diff
@@ -30,11 +31,17 @@ func Playbook(dir, inventory, extraVars, playbook string) (int, error) {
 	c.Stdout = os.Stdout
 	c.Stderr = os.Stderr
 	if err := c.Run(); err != nil {
+		// Try to get the exit code
 		if exitError, ok := err.(*goexec.ExitError); ok {
 			return exitError.ExitCode(), err
+		} else {
+			// This will happen if ansible is not available in $PATH
+			return defaultFailedCode, err
```

Comment:
You can directly mention the non zero here.. no need to declare a variable for that..